### ISSUE TITLE
Warning when overriding properties within the same selector.

### ISSUE THREAD

**Issue by mhemesath** [CSSLint/csslint#51]
Properties that override other properties within the same selector should throw a warning. I'm not sure if this applies for EVERY case, but definitely for cases such as: 

```
.foo { color: #000; display: inline; color: #FFF; } 
```
